### ISSUE TITLE
Error about CytoTRACEKernel

### ISSUE THREAD

**Issue by CenviaTZC** [theislab/cellrank#1088]
Hi, I am following the CellRank beyond RNA velocity tutorial to process my data, and when I run CytoTRACEKernel(adata) step, I get an error as shown below，there are four new columns in adata.obs as shown in the figure, how can I solve this error?Thank you very much!
![image](https://github.com/theislab/cellrank/assets/82918921/5667b5da-0fb6-4768-b01c-62e7d205192c)

```python
ctk = CytoTRACEKernel(adata)
```

```pytb
...Computing CytoTRACE score with `9044` genes. Consider using more than `10000` genes
Adding `adata.obs['ct_score']`
       `adata.obs['ct_pseudotime']`
       `adata.obs['ct_num_exp_genes']`
       `adata.var['ct_gene_corr']`
       `adata.var['ct_correlates']`
       `adata.uns['ct_params']`
    Finish (0:00:00)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[206], line 1
----> 1 CytoTRACEKernel(adata)

File ~\.conda\envs\Tangram_py38\lib\site-packages\cellrank\tl\kernels\_cytotrace_kernel.py:95, in CytoTRACEKernel.__init__(self, adata, backward, layer, aggregation, use_raw, compute_cond_num, check_connectivity, **kwargs)
     82 def __init__(
     83     self,
     84     adata: AnnData,
   (...)
     93     **kwargs: Any,
     94 ):
---> 95     super().__init__(
     96         adata,
     97         backward=backward,
     98         time_key=Key.cytotrace("pseudotime"),
     99         compute_cond_num=compute_cond_num,
    100         check_connectivity=check_connectivity,
    101         layer=layer,
    102         aggregation=aggregation,
    103         use_raw=use_raw,
    104         **kwargs,
    105     )
    106     self._time_key = Key.cytotrace("pseudotime")

File ~\.conda\envs\Tangram_py38\lib\site-packages\cellrank\tl\kernels\_pseudotime_kernel.py:58, in PseudotimeKernel.__init__(self, adata, backward, time_key, compute_cond_num, check_connectivity, **kwargs)
     49 def __init__(
     50     self,
     51     adata: AnnData,
   (...)
     56     **kwargs: Any,
     57 ):
---> 58     super().__init__(
     59         adata,
     60         backward=backward,
     61         time_key=time_key,
     62         compute_cond_num=compute_cond_num,
     63         check_connectivity=check_connectivity,
     64         **kwargs,
     65     )
     66     self._time_key = time_key

File ~\.conda\envs\Tangram_py38\lib\site-packages\cellrank\tl\kernels\_base_kernel.py:965, in Kernel.__init__(self, adata, backward, compute_cond_num, check_connectivity, **kwargs)
    954 def __init__(
    955     self,
    956     adata: AnnData,
   (...)
    960     **kwargs: Any,
    961 ):
    962     super().__init__(
    963         adata, backward, op_name=None, compute_cond_num=compute_cond_num, **kwargs
    964     )
--> 965     self._read_from_adata(check_connectivity=check_connectivity, **kwargs)

File ~\.conda\envs\Tangram_py38\lib\site-packages\cellrank\tl\kernels\_cytotrace_kernel.py:120, in CytoTRACEKernel._read_from_adata(self, time_key, layer, aggregation, use_raw, **kwargs)
    108 def _read_from_adata(
    109     self,
    110     time_key: str,
   (...)
    116     **kwargs: Any,
    117 ) -> None:
    118     self.compute_cytotrace(layer=layer, aggregation=aggregation, use_raw=use_raw)
--> 120     super()._read_from_adata(time_key=time_key, **kwargs)

File ~\.conda\envs\Tangram_py38\lib\site-packages\cellrank\tl\kernels\_pseudotime_kernel.py:79, in PseudotimeKernel._read_from_adata(self, time_key, **kwargs)
     76     self._pseudotime = np.max(self.pseudotime) - self.pseudotime
     78 if np.any(np.isnan(self._pseudotime)):
---> 79     raise ValueError("Encountered NaN values in pseudotime.")

ValueError: Encountered NaN values in pseudotime.

```

#### Versions:

> ...cellrank==1.5.1 scanpy==1.9.3 anndata==0.9.1 numpy==1.22.4 numba==0.55.2 scipy==1.10.1 pandas==1.5.3 pygpcca==1.0.4 scikit-learn==1.2.2 statsmodels==0.13.5 python-igraph==0.10.4 scvelo==0.2.5 pygam==0.8.0 matplotlib==3.7.1 seaborn==0.12.2

**Comment by Marius1311** [comment]
Ah, I think that's a known bug that can be resolved by installing the latest CellRank version from main and following the updated CytoTRACE tutorial here: https://cellrank.readthedocs.io/en/latest/notebooks/tutorials/kernels/400_cytotrace.html

**Comment by michalk8** [comment]
Should be solved in 2.0, closing this.
